about/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: About the LLNL Software Portal
+layout: default
+---
+
+## {{ page.title }}
+{: .page-header}
+
+Welcome to the Lawrence Livermore National Laboratory software portal! The purpose of this software portal (or catalog if you prefer) is to serve as the definitive source of truth pointing to all software that has been produced and released by Lawrence Livermore National Laboratory.
+
+LLNL produces software on a daily basis. Some of this software is used only internally, other components are licensed for use by external partners and collaborators, still other software is released, or even actively developed, in the open on software hosting platforms such as GitHub.com, Bitbucket.org, Sourceforge.net, and others.
+
+This page is designed to be a single stop to find all types of software produced and available due to the expertise of Lawrence Livermore National Laboratory scientists and engineers.
```

Comment:
This sentence seems redundant when taken with paragraph 1.  Do we have some other language we could use that highlights LLNL's enthusiasm for OSS?